pkg/virt-api/api.go

```diff
@@ -611,7 +604,7 @@ func (app *virtAPIApp) composeSubresources() {
 	ws.Route(ws.GET("/").
 		Produces(restful.MIME_JSON).Writes(metav1.RootPaths{}).
 		To(func(request *restful.Request, response *restful.Response) {
-			paths := []string{"/apis",
+			paths := []string{
```

Comment:
Have you checked if there is any difference between `/apis` and `/apis/` endpoints?
If there is no difference, can you instead keep the one without trailing slash?

Reply:
Right, we should keep the one without the trailing slash. We only add one endpoint without a trailing slash, so it should make no difference. I tested it and it was working with and without trailing slash when only adding the `/apis` endpoint. The list of `nonAuthEndpoints` needed to adjusted although.

https://github.com/kubevirt/kubevirt/blob/8a795fc870524e36ce992d355e2119a7f6aa3a4a/pkg/virt-api/api.go#L648

A call to `/` on the Kubernets API (`kubectl proxy) lists `/apis` and `/apis`, but the reason for that looks to be a path matching issue with the gorestful library used in the Kubernetes API server. I suspect we don't need to mimic this behavior in our code.

See these comments/code:

https://github.com/kubernetes/kubernetes/blob/2f7fae34514991e43274336f5ca251b19c03c629/staging/src/k8s.io/apiserver/pkg/server/handler.go#L53
https://github.com/kubernetes/kubernetes/blob/2f7fae34514991e43274336f5ca251b19c03c629/staging/src/k8s.io/apiserver/pkg/server/handler.go#L129